go/pam_script_ses_open/main.go

```diff
@@ -140,8 +140,8 @@ func initCgroup() {
 		}
 	}
 
-	// 512MiB
-	const memoryLimit = 512 * 1024 * 1024
+	// 4 GiB
+	const memoryLimit = 4 * 1024 * 1024 * 1024
```

Comment:
I note that this fails to compile on a 32-bit system, but I'm okay with that (because it actually fails to compile, rather than fails silently).